stubs/EloquentBuilder.stub

```diff
@@ -437,7 +437,7 @@ class Builder
      *
      * @param  mixed  $value
      * @param  callable($this, mixed): (void|Builder<TModelClass>)  $callback
-     * @param  callable($this, mixed): (null|Builder<TModelClass>)|null  $default
+     * @param  callable($this, mixed): (null|void|Builder<TModelClass>)|null  $default
```

Comment:
Isn't it void in place of null?
```suggestion
     * @param  callable($this, mixed): (void|Builder<TModelClass>)|null  $default
```

Reply:
Yeah, I was thinking the same. But might be a breaking change if people are already return null now. Let me know If I need to change it.

Reply:
@canvural I'm out. The framework uses type juggling: `elseif ($default)`.
Please lead the way!

Reply:
@BertvanHoekelen In PHP reality void is really null 😢 
```php
function noVoidInPhp(): void
{
    echo 'In PHP reality void is really: ';
}

var_dump(noVoidInPhp());
```

Reply:
Sorry, I'm paralyzed by code-liberalism :)

Reply:
In this case I think it's better to have something like `(callable($this, mixed): Builder<TModelClass>)|(callable($this, mixed): void)` to say "this callable can return either `void` or `Builder`"

Can you try that? Thanks.

Reply:
@canvural Done 💪🏼